ui/app/styles/components/button.scss

```diff
@@ -92,11 +86,23 @@ a[href].button {
   }
 
   &--external-link {
-    > .pds-icon {
+    > svg {
```

Comment:
here you're relying on the component internal structure (expects the SVG to be a direct child) and in case we decide in the future to change it, this would break (unknowingly) the styling.

my suggestion is to use a wrapping container/div, and style that container. in this way you decouple hosting codebase from the components implementation